Makefile

```diff
@@ -42,3 +42,17 @@ cross-images:
 		docker build --tag quickwitinc/builder-$$tag --file ./build/cross-images/$$tag.dockerfile ./build/cross-images; \
 		docker push quickwitinc/builder-$$tag:latest; \
 	done
+
+
+# TODO: To be replaced by https://github.com/quickwit-inc/quickwit/issues/237
+.PHONY: build-x86_64-unknown-linux-gnu
```

Comment:
I think I'd like `make build ARCH=x86_64-unknown-linux-gnu` better? The default value could be that of the hosts' arch. WDYT?

Reply:
I think this is better, because I also felt someone always has to look this up and since this can be a good default `ARCH=x86_64-unknown-linux-gnu` let's go your option

Reply:
@guilload  I settled with `TARGET` instead of `ARCH` as the former is what we are actually referring to here.

Reply:
Yeah, `TARGET` is great.